examples/server/server.c

```diff
@@ -225,6 +229,28 @@ static WC_INLINE int PeekSeq(const char* buf, word32* seq)
     return 0;
 }
 
+#ifdef HAVE_LIBOQS
+int liboqs_nametogroup(const char* name)
```

Comment:
`static`?

Reply:
going to remove it....dead code....never gets called :)